components/gitpod-cli/cmd/credential-helper.go

```diff
@@ -54,13 +54,29 @@ var credentialHelper = &cobra.Command{
 			fmt.Printf("username=%s\npassword=%s\n", user, token)
```

Comment:
Without stdout, git will ask user to input their username and password.
Is this better? or it will cause security problem?